crates/biome_css_analyze/src/keywords.rs

```diff
@@ -81,3 +81,683 @@ pub const FONT_SIZE_KEYWORDS: [&str; 9] = [
 ];
 
 pub const LINE_HEIGHT_KEYWORDS: [&str; 1] = ["normal"];
+
+/// List of known CSS value functions sourced from [`css-functions-list`](https://www.npmjs.com/package/css-functions-list).
+/// See the original list [here](https://github.com/niksy/css-functions-list/blob/master/index.json).
+pub const FUNCTION_KEYWORDS: [&str; 675] = [
+    "abs",
+    "acos",
+    "anchor",
+    "anchor-size",
+    "annotation",
+    "asin",
+    "atan",
+    "atan2",
+    "attr",
+    "blur",
+    "brightness",
+    "calc",
+    "character-variant",
+    "circle",
+    "clamp",
+    "color",
+    "color-contrast",
+    "color-mix",
+    "color-stop",
+    "conic-gradient",
+    "contrast",
+    "cos",
+    "counter",
+    "counters",
+    "cross-fade",
+    "cubic-bezier",
+    "device-cmyk",
+    "drop-shadow",
+    "element",
+    "ellipse",
+    "env",
+    "exp",
+    "fit-content",
+    "format",
+    "from",
+    "gradient",
+    "grayscale",
+    "hsl",
+    "hsla",
+    "hue-rotate",
+    "hwb",
+    "hypot",
+    "image",
+    "image-rect",
```

Comment:
You can remove this one, `-o-image-rect`, `-ms-image-rect`, `-o-image-rect` and `-webkit-image-rect`.
It's the only non-prefixed function name which is bogus AFAIK.
Keep`-moz-image-rect` though.

https://caniuse.com/mdn-css_properties_background-image_image-rect
https://bugs.webkit.org/show_bug.cgi?id=32177

Reply:
I have removed them.